editor/spatial_editor_gizmos.cpp

```diff
@@ -2419,7 +2430,7 @@ void ParticlesGizmo::redraw() {
 
 		gizmo_color.a = 0.1;
 		Ref<Material> solid_material = create_material("particles_solid_material", gizmo_color);
-		add_solid_box(solid_material, aabb.get_size());
+		add_solid_box(solid_material, aabb.get_size(), aabb.get_position() + aabb.get_size()/2.0);
```

Comment:
Style issue here as there should be a space on each side of the `/` operator. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Sorry for the inconvinience, it should be ok now. I have set up the formatting git hook.